src/components/CalendarMonth.jsx

```diff
@@ -182,7 +183,11 @@ class CalendarMonth extends React.Component {
           <strong>{monthTitle}</strong>
         </div>
 
-        <table {...css(styles.CalendarMonth_table)} role="presentation">
+        <table
+          {...css(styles.CalendarMonth_table)}
+          role="presentation"
+          aria-labelledby={`CalendarMonth_caption__${toISOMonthString(month)}`}
```

Comment:
since this has `role="presentation"`, the label here doesn't mean as much (and also possibly won't even get read)

Reply:
hmmm, does this belong in the DayPicker then? hmmm. But the DayPicker contains all the months, so perhaps it needs a different label? 

Trying to follow up on https://github.com/airbnb/react-dates/pull/715/files#r139019568

Reply:
I don't know that it's necessary, and I think even in general case, I don't know that the month name is as meaningful as a label outside of this table. I would imagine that the container for the picker may benefit from a label like "Date picker" or "Select a date", but the month is less valuable information for the higher level container

Reply:
I'll just kill the id then